Automattic/jetpack
# Comment: Protect: Add Logo component

Fixes #

#### Changes proposed in this Pull Request:

* Protect: Add Logo component

#### Jetpack product discussion


p1651582619211719-slack-C02TQF5VAJD

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with Storybook
* Plugins / Protect / Logo

https://user-images.githubusercontent.com/77539/166493157-baab2b92-5735-4e9a-9041-553f38d43e54.mov




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202217068684590
